### PR TITLE
Fixing PEP8 build, and opt in using ci-helpers

### PR DESCRIPTION
**Pull request by bsipocz**
This PR fixes the pep8 travis build (that never actually run before), and opt in using ``ci-helpers``.

### CODE REVIEW

**Comment by bsipocz** [comment]
Also I wonder whether you want to put py 3.5 and numpy 1.10 into the matrix. 
Obviously the same 3 tests as for numpy dev are failing, but there's nothing else on top of those.

https://travis-ci.org/bsipocz/astroplan/builds/92838729

**Comment by bmorris3** [comment]
Thanks @bsipocz. Would you rebase against master?

**Comment by cdeil** [comment]
Yes, I think adding a Python 3.5 and Numpy 1.10 build to the test matrix is a good idea.

**Comment by bsipocz** [comment]
Rebased and added py3.5 and np1.10.

**Comment by cdeil** [comment]
The `test_sun_separation` fail is in master and unreleated:
https://travis-ci.org/astropy/astroplan/jobs/93785851#L475
@bsipocz Could you please add `@pytest.mark.xfail` in this PR as a temp fix?

File: .travis.yml [hunk 1]
```diff
@@ -39,57 +41,86 @@ matrix:
...
         # Try all python versions with the latest numpy
         # TODO: add the `--open-files` option back once this issue has been resolved:
         # https://github.com/astropy/astroplan/pull/83#issuecomment-136129489
         - os: linux
           env: PYTHON_VERSION=2.7 SETUP_CMD='test -V'
+               CONDA_DEPENDENCIES='pytz'
+               PIP_DEPENDENCIES=''
+
+        # There isn't numpy 1.10 and python 3.3 combination in conda or
+        # astropy-ci-extras, yet. Update this when it becomes available
```
Comment (cdeil) [inline .travis.yml]: Please remove the "update this when it becomes available".

There's very few Python 3 users on Python 3.3:
http://astrofrog.github.io/blog/2015/05/09/2015-survey-results/
It's OK to keep 3.3 in the build matrix as long as it doesn't generate extra work to support, but we don't have to add more Python 3.3 builds in the future.

File: .travis.yml [hunk 2]
```diff
@@ -97,29 +128,42 @@ matrix:
...
-    - if [[ $MAIN_CMD == 'python setup.py' ]]; then
-        $MAIN_CMD $SETUP_CMD;
+             CONDA_DEPENDENCIES='pytz'
+             PIP_DEPENDENCIES=''
+
+before_install:
+    - git clone git://github.com/astropy/ci-helpers.git
+    - source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh
+
+    # This is needed to make matplotlib plot testing work
```
Comment (cdeil) [inline .travis.yml]: Many packages need this. Move this to `ci-helpers` and make it opt-in, i.e. execute these commands if the user sets something like `SETUP_DISPLAY_FOR_MATPLOTLIB`?

**Comment by cdeil** [comment]
@bsipocz This looks good except that the one test needs to be xfailed.
Please ping me when travis-ci gives green light and I'll merge.

File: .travis.yml (hunk 2 above)
Comment (bsipocz) [inline .travis.yml]: I've opened an issue for this in astropy/ci-helpers#26

File: .travis.yml (hunk 1 above)
Comment (bsipocz) [inline .travis.yml]: Done. However I didn't mean to add a new build, but to change numpy_version to 1.10 when/if this combination becomes available.

**Comment by bsipocz** [comment]
@cdeil - What about ``sun_set_time`` and ``sun_rise_time`` docstring fails?

**Comment by cdeil** [comment]
> What about sun_set_time and sun_rise_time docstring fails?

Skip them here so that this can be merged.
And please file a separate issue so that those are not forgotten.
(there might already be an issue / pull request .... I've lost track ... if it's not clear just open a new one)

**Comment by bsipocz** [comment]
Now there is a ``remote_data`` related fail here: https://travis-ci.org/astropy/astroplan/jobs/93924145

I just wonder, wouldn't it be better to move the ``remote_data`` build in the allowed to fail lot, thus keeping the travis result independent of some unrelated random timeouts?

**Comment by bmorris3** [comment]
Historically we've been pinging each other to restart timed-out travis runs. I've restarted this one.

**Comment by cdeil** [comment]
@bsipocz - Thank you!

**Pull request merged**